src/extension.ts

```diff
@@ -24,4 +24,24 @@ export async function activate(context: vscode.ExtensionContext) {
       }));
 
   await clangdContext.activate(context.globalStoragePath, outputChannel);
+
+  setInterval(function() {
+    const cpptools = vscode.extensions.getExtension('ms-vscode.cpptools');
+    if (cpptools !== undefined && cpptools.isActive) {
```

Comment:
nit: maybe just cppTools && cppTools.isActive? (the docs say it'll be undefined rather than null but...)